stdlib/mexpr/pprint.mc

```diff
@@ -107,15 +107,15 @@ let _parserStr = lam str. lam prefix. lam cond.
 -- overriding them.
 
 -- Constructor string parser translation
-let pprintConString = lam str.
+let _pprintConString = lam str.
```

Comment:
Is there a reason not to inline these in the `IdentifierPrettyPrint` fragment?

Reply:
Only laziness, fixed.